WordPress/WordPressScreenshotGeneration/WordPressScreenshotGeneration.swift

```diff
@@ -89,7 +109,7 @@ class WordPressScreenshotGeneration: XCTestCase {
         if UIDevice.current.userInterfaceIdiom == .pad {
             blogDetailsTable.cells["Blog Post Row"].tap()
             waitForElementToExist(element: app.tables["PostsTable"])
-            sleep(5) // Wait for posts to load
+            sleep(imagesWaitTime) // Wait for post images to load
```

Comment:
While in general, I don't like tests to rely on `sleep` and I'd prefer some other solution (like monitoring network activity or something like that), this is just a test to drive screenshots, so I agree that we can live happy with this solution for this purpose.

Reply:
Yes I agree completely. This may be something worth tackling in the future, but it makes sense to keep like this for now.